app/controlplane/api/workflowcontract/v1/crafting_schema.pb.go

```diff
@@ -43,6 +43,7 @@ const (
 	CraftingSchema_Runner_GITHUB_ACTION           CraftingSchema_Runner_RunnerType = 1
 	CraftingSchema_Runner_GITLAB_PIPELINE         CraftingSchema_Runner_RunnerType = 2
 	CraftingSchema_Runner_AZURE_PIPELINE          CraftingSchema_Runner_RunnerType = 3
+	CraftingSchema_Runner_JENKINS_JOB	      CraftingSchema_Runner_RunnerType = 4
```

Comment:
It seems that you've manually updated this value when in fact this is the result of modifying crafting_schema.proto

https://github.com/chainloop-dev/chainloop/blob/01432f04375e2d6a25084fe88c29eff1c32f5da4/app/controlplane/api/workflowcontract/v1/crafting_schema.proto#L42-L46

and then running `make api`. 

The good news is that I believe you've found a bug, for some reason our CI has not detected this, and it should have. We have similar checks for other generated code but not this one interestingly.

Just for runs, if you run `make api` in the code of your patch, you'll see how this line gets removed

```diff
diff --git a/app/controlplane/api/workflowcontract/v1/crafting_schema.pb.go b/app/controlplane/api/workflowcontract/v1/crafting_schema.pb.go
index 3de6c8f..0b782cf 100644
--- a/app/controlplane/api/workflowcontract/v1/crafting_schema.pb.go
+++ b/app/controlplane/api/workflowcontract/v1/crafting_schema.pb.go
@@ -43,7 +43,6 @@ const (
        CraftingSchema_Runner_GITHUB_ACTION           CraftingSchema_Runner_RunnerType = 1
        CraftingSchema_Runner_GITLAB_PIPELINE         CraftingSchema_Runner_RunnerType = 2
        CraftingSchema_Runner_AZURE_PIPELINE          CraftingSchema_Runner_RunnerType = 3
-       CraftingSchema_Runner_JENKINS_JOB             CraftingSchema_Runner_RunnerType = 4
 )

```

So as I said, instead, modify the proto definition and run the generation, that should persist it.

Thanks!

Reply:
Ohh interesting!
I bet it's something I know, like some knowledge I am missing, because I didn't realize at all that file was generated.
My bad my bad.
Will fix it!
Thanks

Reply:
as I said, the CI should have warned us, I'll fix that :)

Reply:
@buccarel, this change https://github.com/chainloop-dev/chainloop/pull/412 is what was missing in the CI to detect your use-case.

If we happen to merge it and your code rebased, your PR should then fail telling you that you are not committing all the generated code or that it differs from the baseline.